src/ImageSharp/Colors/PackedPixel/IPixel.cs

```diff
@@ -0,0 +1,68 @@
+﻿namespace ImageSharp
+{
+    using System;
+    using System.Numerics;
+
+    public interface IPixel
```

Comment:
We don't use the property but I'm sure Monogame must. We, of course,  don't want to implement it for our optimized types nor use the interface for typeparams. Time to check the Monogamr source.